src/tasks.py

```diff
@@ -286,6 +286,32 @@ def get_depth_by_time_at_point(model_id: int, lat: float, lng: float) -> DepthTi
     return DepthTimePlot(depths, times)
 
 
+@app.task(base=OnFailureStateTask)
+def retrieve_medusa_input_parameters(scenario_id: int) -> Dict[str, Union[str, float]]:
```

Comment:
this could also return `None`, so it needs to be wrapped in an `Optional[ ]` so that its clear. It would be good to describe in the docstring what `None` means

Reply:
I have added `Optional[ ]` in.